src/pages/midwife/homeVisit.jsx

```
@@ -0,0 +1,175 @@
+      }
+    };
+    fetchParentDataForMidwife();
+    return () => {
+      isComponentMounted = false;
+    };
+  }, [userId, navigate]);
+
+  const isValidLocation =
+    location?.lat &&
+    location?.lng &&
+    !isNaN(location.lat) &&
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)